### PR TITLE
created the volunteer form page

### LINKED ISSUE #70: Volunteer Form Page
image

Follow this guide: https://blog.webjeda.com/google-form-customize/
Look at the waiver form for an example
Form: https://docs.google.com/forms/d/e/1FAIpQLScItUJcJ7aH6Iz8eLC0bxpzxbRr4u5trZeou8Uu5z2qLLSRTQ/viewform

## Requirements:
1. Use `FormHeader` component
2. Use `FormButton` component
3. Go to `/volunteer`

### PR DESCRIPTION
**Pull request by kayliezhao**
created the volunteer form page
image
image

### CODE REVIEW

**Comment by kayliezhao** [comment]
linting check isn't passing even though i did npm run eslint

File: src/app/volunteer/page.tsx [hunk 1]
```diff
@@ -1,8 +1,176 @@
...
+          <div>
+            <label className="mb-1 block font-archivo-black text-lg font-bold uppercase md:text-2xl lg:text-3xl">
+              Do you have a way to get to RBA?*
+            </label>
+            {/* first option */}
+            <div className="flex hover:bg-gray-700">
+              <input
+                name="entry.111141012"
+                type="radio"
+                id="yess"
```
Comment (SafeDuck) [inline src/app/volunteer/page.tsx]: choose a different id

File: src/app/volunteer/page.tsx [hunk 2]
```diff
@@ -1,8 +1,176 @@
...
+                placeholder="option1"
+                className="h-[10] w-4 border-x-hlg-red-300"
+                required
+              />
+              <label
+                htmlFor="yess"
```
Comment (SafeDuck) [inline src/app/volunteer/page.tsx]: choose a different htmlFor

File: src/app/volunteer/page.tsx [hunk 3]
```diff
@@ -1,8 +1,176 @@
...
+import logo from "@/public/logo.webp";
+import Image from "next/image";
+
 const page = () => {
+  const [isSubmitted, setIsSubmitted] = useState(false);
+
+  if (isSubmitted) {
+    return (
+      <div className="flex min-h-screen w-full flex-col items-center justify-center gap-5 font-archivo-black text-4xl">
+        {/* <FormHeader txt="WAIVER SUBMIITED" /> */}
```
Comment (SafeDuck) [inline src/app/volunteer/page.tsx]: remove comment

File: src/app/volunteer/page.tsx [hunk 4]
```diff
@@ -1,8 +1,176 @@
...
+import FormHeader from "@/components/home/FormHeader";
+import FormButton from "@/components/FormButton";
+import logo from "@/public/logo.webp";
+import Image from "next/image";
+
 const page = () => {
+  const [isSubmitted, setIsSubmitted] = useState(false);
+
+  if (isSubmitted) {
+    return (
```
Comment (SafeDuck) [inline src/app/volunteer/page.tsx:20]: You need to add all of this:
image

**Review by SafeDuck — CHANGES_REQUESTED** [review]

**Comment by kayliezhao** [comment]
added missing description
image
image

**Review by SafeDuck — APPROVED** [review]

**Pull request merged**